validator/server_arb/validator_spawner.go

```diff
@@ -11,32 +11,33 @@ import (
 	"sync/atomic"
 	"time"
 
-	"github.com/offchainlabs/nitro/util/readymarker"
+	flag "github.com/spf13/pflag"
+
+	"github.com/offchainlabs/nitro/util/containers"
 	"github.com/offchainlabs/nitro/util/stopwaiter"
 	"github.com/offchainlabs/nitro/validator"
 	"github.com/offchainlabs/nitro/validator/server_common"
-	flag "github.com/spf13/pflag"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/log"
 )
 
 type ArbitratorSpawnerConfig struct {
-	ConcurrentRuns int                `koanf:"concurrent-runs-limit" reload:"hot"`
-	OutputPath     string             `koanf:"output-path" reload:"hot"`
-	Execution      MachineCacheConfig `koanf:"execution" reload:"hot"` // hot reloading for new executions only
+	Workers    int                `koanf:"workers" reload:"hot"`
+	OutputPath string             `koanf:"output-path" reload:"hot"`
+	Execution  MachineCacheConfig `koanf:"execution" reload:"hot"` // hot reloading for new executions only
 }
 
 type ArbitratorSpawnerConfigFecher func() *ArbitratorSpawnerConfig
 
 var DefaultArbitratorSpawnerConfig = ArbitratorSpawnerConfig{
-	ConcurrentRuns: 0,
-	OutputPath:     "./target/output",
-	Execution:      DefaultMachineCacheConfig,
+	Workers:    0,
+	OutputPath: "./target/output",
+	Execution:  DefaultMachineCacheConfig,
 }
 
 func ArbitratorSpawnerConfigAddOptions(prefix string, f *flag.FlagSet) {
-	f.Int(prefix+".concurrent-runs-limit", DefaultArbitratorSpawnerConfig.ConcurrentRuns, "number of cuncurrent runs")
+	f.Int(prefix+".workers", DefaultArbitratorSpawnerConfig.Workers, "number of cuncurrent validation runs")
```

Comment:
cuncurrent -> concurrent . Maybe add " (number of threads/cores to use)" to the end too.